src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -70,6 +70,8 @@ public async Task<IActionResult> GetHostStatus([FromServices] IScriptHostManager
                 State = scriptHostManager.State.ToString(),
                 Version = ScriptHost.Version,
                 VersionDetails = Utility.GetInformationalVersion(typeof(ScriptHost)),
+                PlatformVersion = _environment.GetAntaresVersion(),
+                InstanceId = _environment.GetEnvironmentVariableOrDefault(EnvironmentSettingNames.MachineIdentifier, string.Empty),
```

Comment:
Since MachineIdentifier will always be empty on Linux consumption, can we fall back to
Environment.GetEnvironmentVariable(ContainerNameKey)

Reply:
Updated to use Environment.GetEnvironmentVariable(ContainerName) for linux consumption.